jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshReportType.java

```diff
@@ -0,0 +1,10 @@
+package org.jbehavesupport.core.ssh;
+
+/**
+ *Modes for ServerLogXmlReporterExtension
```

Comment:
```suggestion
 * Modes for ServerLogXmlReporterExtension
```
Might be worth to even put a `{@link ServerLogXmlReporterExtension}` with fully qualified name